File: JS/Solution_Day_1.js
let addUp = (arr, k) => {
  for(element1 of arr){
    
    for(element2 of arr){
      if(element1 + element2 === k){
        return true;
      }
    }
    
  }
  
  return false;
}

let n, k;

n = parseInt(window.prompt("Input numbers of element: "));
let arr = [];

alert(`Input ${n} numbers : `);

for(let i = 0; i < n; i++){
  arr[i] = parseInt(window.prompt(`Number ${i + 1} : `));
}

k = parseInt(window.prompt("Input a k number : "));

console.log(addUp(arr, k));

File: CS/Solution_Day_1.cs
using System;

namespace MyApp{
	class Program{
		
		static void Main(){
			Console.Write("Numbers in an array : ");
			int n = Convert.ToInt32(Console.ReadLine());
			Console.Write("Input {n} numbers seperate by space : ");
			string[] str = Console.ReadLine().Split(' ');
			Console.Write("Input a k number : ");
			int k = Convert.ToInt16(Console.ReadLine());
			int[] arr = new int[n];
			for(int i = 0; i < n; i++){
				arr[i] = Int32.Parse(str[i]);
			}
			
			Console.WriteLine(AlwaysTrue(arr, k));
		}
		
		static bool AlwaysTrue(int[] array, int num){
			foreach(int element1 in array){
				foreach(int element2 in array){
					if(element1 != element2){
						if(element1 + element2 == num){
							return true;
						}
					}
				}
			}
			return false;
		}
	}
}